### DISCUSSION TITLE
Aspects visible on the opportunity dashboard?

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#560] [Ideas] [answered]
I would really like to see the Aspects on the opportunity dashboard (currently you have to select the Contribute tab in order to browse and create Aspects). I think it would help members to find relevant information about the opportunity more quickly.

**Comment by larssondenise** [comment]
Hi Gerlof, do you mean specifically for opportunities or aspects under the contribute tab in general (so also under challenges and hubs)?

**Reply by GerlofdeHaan** [reply]
In this case: specifically for opportunities

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof, we are adding a preview section for Aspects, and will be making them visible on the dashboards: https://github.com/alkem-io/client-web/issues/1940 - this issue will shortly be updated.

**Reply by techsmyth** [reply]
Raised as https://github.com/alkem-io/server/issues/1876 to sort the opportunities alphabetically - should be in near term

**Reply by GerlofdeHaan** [reply]
Preview section for Aspects on the Opportunities dashboard works great!

**Reply by techsmyth** [reply]
:)
